bundles/paikkatietoikkuna/inspire/Flyout.js

```diff
@@ -9,36 +9,30 @@ Oskari.clazz.define('Oskari.inspire.Flyout',
         this.loc = Oskari.getMsg.bind(null, 'inspire');
         this.container = null;
         this.flyout = null;
-        this.flyoutContent = this.loc('flyoutContent').content;
     }, {
+        __name: 'Oskari.inspire.Flyout',
         getName: function () {
-            return 'Oskari.inspire.Flyout';
+            return this.__name;
         },
         getTitle: function () {
-            return this.loc('flyout.title');
+            return this.instance.loc('flyout.title');
```

Comment:
We might not need instance to have loc on its own. Try removing it and just use the one in the flyout